lib/networking/kustomization.yaml

```diff
@@ -242,15 +242,6 @@ replacements:
       kind: NetConfig
     fieldPaths:
     - spec.networks.[name=Storage].dnsDomain
-- source:
```

Comment:
ack, I assume this is added back in the output via the update in va/hci/kustomization.yaml

Reply:
Correct.  My thinking here was that the base `lib` templates should only have common elements that all DTs/VAs would require.  Thus I removed `StorageMgmt` stuff and explicitly added it to the HCI `kustomization.yaml`.  We can discuss this further if we think there is perhaps a better approach (or if you think the old way was preferable).